<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AcceptRemontWindow</class>
 <widget class="QDialog" name="AcceptRemontWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Принять в ОСО</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Рекламация</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Причина</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Действие</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Дефект</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="deDate">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>88</y>
     <width>230</width>
     <height>25</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="leAction">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>230</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leDefect">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>230</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbExchange">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Произвести замену</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbApply">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Принять в ОСО</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbReason">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>230</width>
     <height>24</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="cbClaim">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>230</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>56</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Устройство</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbDevice">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>52</y>
     <width>230</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
